src/main/java/teammates/ui/controller/CourseJoinAuthenticatedAbstractAction.java

```diff
@@ -0,0 +1,25 @@
+package teammates.ui.controller;
+
+import teammates.common.datatransfer.attributes.CourseAttributes;
+import teammates.common.exception.EmailSendingException;
+import teammates.common.exception.TeammatesException;
+import teammates.common.util.EmailWrapper;
+import teammates.common.util.Logger;
+import teammates.logic.api.EmailGenerator;
+
+public abstract class CourseJoinAuthenticatedAbstractAction extends Action {
+
+    private static final Logger log = Logger.getLogger();
+
+    protected void sendCourseRegisteredEmail(String courseId) {
+        CourseAttributes course = logic.getCourse(courseId);
+        EmailWrapper email = new EmailGenerator().generateUserCourseRegisterEmail(
+                logic.getAccount(account.googleId), course);
+        try {
+            emailSender.sendEmail(email);
+        } catch (EmailSendingException e) {
+            log.severe("Instructor course register email failed to send: " + TeammatesException.toStringWithStackTrace(e));
```

Comment:
> Also, this may need to be more clearly distinguished from the invitation email asking a user to sign up.

@chowyb Just a clarification: by _this_ you meant the log message or the email itself?